orttraining/orttraining/python/ort_trainer.py

```diff
@@ -321,7 +265,7 @@ def convert_model_loss_fn_to_onnx(model, loss_fn, model_desc, device, inputs, op
                        output_names=output_names,
                        opset_version=opset_version,
                        dynamic_axes=dynamic_axes,
-                       training=True,
+                       training=torch.onnx.TrainingMode.TRAINING,
```

Comment:
Are we supporting older pytorch? This will not work and will lead to silent errors. On my tests, I forgot to change that and the script run and the numbers didnt match. Super tricky issue as it doesn't raise any exception

Reply:
exactly, this will be fixed similar to `enable_onnx_checker` flag depending on pytorch version.